pkg/params/validators.go

```diff
@@ -137,7 +137,7 @@ func ValidateDuration(value string) error {
 }
 
 func ValidateIP(value string) error {
-	if net.ParseIP(value) == nil {
+	if value != "" && net.ParseIP(value) == nil {
```

Comment:
I would additionally add a testcase for that so the wanted behavior is "documented"

Reply:
Even better :D